analyzers/rspec/cs/S2386_c#.html

```diff
@@ -33,7 +33,7 @@ <h2>Exceptions</h2>
       <li> <code>System.Collections.Immutable.IImmutableSet&lt;T&gt;</code> </li>
       <li> <code>System.Collections.Immutable.IImmutableStack&lt;T&gt;</code> </li>
       <li> <code>System.Collections.Immutable.IImmutableQueue&lt;T&gt;</code> </li>
-    </ul> </li>
+    </ul>  </li>
```

Comment:
Strange that we get an extra space here.

Reply:
Source: https://raw.githubusercontent.com/SonarSource/rspec/master/rules/S2386/csharp/rule.adoc

Seems just like the way adoc serializes these nested bulletpoints.